<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Recognition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="process_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>640</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #50C878</string>
    </property>
    <property name="text">
     <string>PROCESS</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="result_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>800</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="plainText">
     <string>Result here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);</string>
    </property>
    <property name="text">
     <string>Image here...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="action_load"/>
   </widget>
   <addaction name="menuLoad"/>
  </widget>
  <action name="action_load">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
